swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
It is not necessary to write the `else` and `return False` indented here. You could actually write this like so: 

```suggestion
        if item not in self.inventory:
            return False
            
        self.inventory.remove(item)
         return item
```

By doing the what is shown above you have less nested code, making your logic more readable and maintainable.